### PR TITLE
109_add_permit_and_rules_pages

### PR DESCRIPTION
**Pull request by Dmac26**
﻿## Summary
Addresses issues:
https://app.zenhub.com/workspaces/open-forest-58f8bbe105a35fad2d275a0c/issues/usdaforestservice/usfs-timber-permitting/109

https://app.zenhub.com/workspaces/open-forest-58f8bbe105a35fad2d275a0c/issues/usdaforestservice/usfs-timber-permitting/152

This PR adds the permit application page along with the rules page.

## This pull request is ready to merge when...
- [x] Feature branch starts with the issue number
- [x] Is connected to its original issue via zenhub 👇
- [ ] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Circle)
- [ ] This code has been reviewed by someone other than the original author

### CODE REVIEW

File: frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+      <app-api-error *ngIf="apiErrors" [errors]="apiErrors"></app-api-error>
+
+      <form [formGroup]="applicationForm" class="usa-form-large" novalidate (ngSubmit)="showRulesForm()">
+          <fieldset>
+            <legend>Permit details</legend>
+            <label class="usa-input">Number of cords <span class="required-fields-asterisk"> *</span></label>
+            <p>Each cord is $5. Minimum of 4 cords.
+              <select class="usa-input-inline" id="number-of-cords" formControlName="numberOfCords" aria-required="true">
+                <option selected="selected" [value]="4">4</option>
```
[the remark this answers is not in the document]
Comment (briandavidson) [reply frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.html:39]: I defer to someone on the product/business side for things like this. @carlsonem who would you say is the go-to person for these kinds of questions right now?

[the remark this answers is not in the document]
Comment (Dmac26) [reply frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.html:39]: @briandavidson  @mtlaney - Per Chris and Melissa's suggestion it sounds like we'll want to put in a little more work on this dropdown. But I imagine we can do that on another branch. I'm going to close this conversation and open a card to address that specific issue. Does that work for you guys?

File: frontend/src/sass/pages/_permit-guidelines.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.permit-guidelines {
+//.permit-guidelines {
```
Comment (mtlaney) [inline frontend/src/sass/pages/_permit-guidelines.scss:1]: Can we clean up / remove any commented code

File: frontend/src/sass/pages/_permit-guidelines.scss [hunk 3]
```diff
@@ -8,7 +8,7 @@
   //     padding-top: 60px;
   //   }
   // }
-}
+//}
```
Comment (mtlaney) [inline frontend/src/sass/pages/_permit-guidelines.scss:12]: Can we clean up / remove any commented code

File: frontend/src/sass/pages/_permit-guidelines.scss [hunk 4]
```diff
@@ -99,10 +99,10 @@ markdown#season-dates-markdown {
   }
   h4 {
     background-color: $fs-green;
-    float: left;
+    // float: left;
     text-transform: none;
```
Comment (mtlaney) [inline frontend/src/sass/pages/_permit-guidelines.scss:103]: Can we clean up / remove any commented code

File: frontend/src/app/firewood/forests/firewood-rules/firewood-permit-rules.component.html [hunk 5]
```diff
@@ -1,8 +1,14 @@
+
+<!-- <form [formGroup]="applicationRulesForm" class="usa-form-large" novalidate> -->
```
Comment (mtlaney) [inline frontend/src/app/firewood/forests/firewood-rules/firewood-permit-rules.component.html:2]: Can we clean up / remove any commented code

File: frontend/src/sass/pages/_permit-guidelines.scss (hunk 2 above)
Comment (Dmac26) [reply frontend/src/sass/pages/_permit-guidelines.scss:1]: Commented code removed

File: frontend/src/sass/pages/_permit-guidelines.scss (hunk 3 above)
Comment (Dmac26) [reply frontend/src/sass/pages/_permit-guidelines.scss:12]: Commented code removed

File: frontend/src/app/firewood/forests/firewood-rules/firewood-permit-rules.component.html (hunk 5 above)
Comment (Dmac26) [reply frontend/src/app/firewood/forests/firewood-rules/firewood-permit-rules.component.html:2]: Commented code removed

**Pull request merged**